### DISCUSSION TITLE
Exception regarding the dependency in intl_phone_number_input

### DISCUSSION THREAD

**Discussion by dilshaddkhan** [natintosh/intl-phone-number-input#200] [Q&A] [answered]
When I have already declare  the below plugins.
   equatable: ^2.0.0
  libphonenumber: ^2.0.0
  intl_phone_number_input: ^0.6.0

then why I am still getting the issue like
"
Because intl_phone_number_input >=0.5.2 depends on libphonenumber ^1.0.2 and dummy_contact_page depends on libphonenumber ^2.0.0, intl_phone_number_input >=0.5.2 is forbidden.
So, because dummy_contact_page depends on intl_phone_number_input ^0.6.0, version solving failed.
pub get failed (1; So, because dummy_contact_page depends on intl_phone_number_input ^0.6.0, version solving failed.)
"

**Comment by natintosh** [comment, marked as answer]
Update the package to the recent one.

```yaml
  intl_phone_number_input: ^0.7.0
```
